lib/recurly.js

```diff
@@ -140,10 +154,27 @@ export class Recurly extends Emitter {
     return storageFetch({ key: 'deviceId', otherwise: uid() });
   }
 
+  set deviceId (id) {
+    return storageSet({ key: 'deviceId', value: id });
+  }
+
   get sessionId () {
     return storageFetch({ scope: 'session', key: 'sessionId', otherwise: uid() });
   }
 
+  set sessionId (id) {
+    return storageSet({ scope: 'session', key: 'sessionId', value: id });
+  }
```

Comment:
These setters were added in order to allow fields to override their request identifiers. This also makes it easier to establish DNT patterns